src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -93,58 +98,38 @@ export class IncidentDetail extends React.Component {
     }
   }
 
-  onThor() {
-    const patch = {
-      id: this.props.id,
-      type: 'thor',
-      patch: {
-        status: {
-          state: 'ready to send',
-          text: 'Te verzenden naar THOR',
-          target_api: 'sigmax',
-        },
-      },
-    };
-
-    this.props.onPatchIncident(patch);
-  }
-
-  onDismissSplitNotification() {
-    this.props.onDismissSplitNotification();
-  }
-
   onShowLocation() {
     this.setState({
       previewState: 'showLocation',
-      attachment: '',
+      attachmentHref: '',
     });
   }
 
   onEditLocation() {
     this.setState({
       previewState: 'editLocation',
-      attachment: '',
+      attachmentHref: ''
     });
   }
 
   onEditStatus() {
     this.setState({
       previewState: 'editStatus',
-      attachment: '',
+      attachmentHref: ''
     });
   }
 
-  onShowAttachment(attachment) {
+  onShowAttachment(attachmentHref) {
     this.setState({
       previewState: 'showImage',
-      attachment,
+      attachmentHref
```

Comment:
Should have a trailing comma

Reply:
dat vind ik geen relevante commentaar.
als je dat wil stel ik voor om in een aparte branch alle trailing commas te maken.